zeebe-operate/templates/ingress.yaml

```diff
@@ -1,16 +1,23 @@
+{{- if .Values.ingress.enabled -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
 apiVersion: extensions/v1beta1
+{{- end }}
 kind: Ingress
 metadata:
-  annotations:
-    kubernetes.io/ingress.class: nginx
-    ingress.kubernetes.io/rewrite-target: "/"
-    nginx.ingress.kubernetes.io/ssl-redirect: "false"    
   name: {{ include "zeebe-operate.fullname" . }}
+  labels: {{ include "zeebe-operate.labels" . | nindent 4 }}
```

Comment:
@PashCracken this are all great changes.. and it great that it is coming from the community!